Keyboard/ViewModels/MainWindowViewModel.cs

```diff
@@ -65,11 +68,37 @@ private void _AddNumber()
                 _Clear();
             }
         }
+        public async void _JsonSerializeNow()
+        {
+            var path = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData) + "\\Keyboard\\NumberList\\";
+            string filePath = path + "NumberList.json";
+
+            if (!Directory.Exists(path))
+            {
+                DirectoryInfo di = Directory.CreateDirectory(path);
+            }
+
+            var options = new JsonSerializerOptions { WriteIndented = true };
 
+            using FileStream createStream = File.Create(filePath);
+            await JsonSerializer.SerializeAsync(createStream, NumberList, options);
```

Comment:
I noticed that this function isn't working fully as intended, sometimes it produces empty file and sometimes halfway written file. We might have to reconsider/test how to avoid that problem. Or simply use .Serialize()